packages/backend/src/scheduler/SchedulerWorker.ts

```diff
@@ -40,9 +40,8 @@ export const getDailyDatesFromCron = (
     return dailyDates;
 };
 
-const workerLogger = new GraphileLogger((scope) => (level, message, meta) => {
-    const sanitizedLevel = level === 'warning' ? 'warn' : level;
-    Logger[sanitizedLevel](message, meta, scope);
+const workerLogger = new GraphileLogger((scope) => (_, message, meta) => {
+    Logger.debug(message, meta, scope);
```

Comment:
We now rely on graphile events to record useful info logs. We can still record these as debug logs for further debugging.